### PR TITLE
Update "installation-and-operations" page with all options to "get" OpenProject

### PR DESCRIPTION
**Pull request by ivangriggs**
Updated as a "Gateway" for "How to 'get' OpenProject" to give all the options. I have used texts from other documentation and from the OpenProject website, but may need OP staff review

[ci skip]

### CODE REVIEW

**Comment by oliverguenther** [comment]
Hi @ML-OpenP , @birthe : Not sure who might feel responsible for this topic, but could we assign a review to someone from the docs content team for this?

Best
Oliver

**Comment by bluedeepimpact** [comment]
Hi @oliverguenther sure, thanks for heads up!

**Review by bluedeepimpact — APPROVED** [review]

**Comment by oliverguenther** [comment]
Thanks @ML-OpenP  :bow:

File: docs/installation-and-operations/README.md [hunk 1]
```diff
@@ -1,25 +1,43 @@
 # Installation and Operations Guides
 
-This section concerns the installation and operation of **OpenProject** (Community and Enterprise on-premises editions) for general project management.
+This page summarises the options for getting OpenProject, some hosted and some on-premise. With this information you should be able to decide what is best for you.
```
Comment (bluedeepimpact) [inline docs/installation-and-operations/README.md:3]: I would change this to
`This section concerns the installation and operation of Community Edition and Enterprise on-premises editions. 
The overview below summarizes the options for getting OpenProject, some hosted and some on-premise. You can find a detailed feature comparison [here](https://www.openproject.org/pricing/#compare).`

**Review by bluedeepimpact — COMMENTED** [review]
As this is already merged I will close this review. Thanks a lot for your contribution, @Amandine-Malo!

**Pull request merged**